.github/workflows/actuated-aarch64-test.yaml

```diff
@@ -0,0 +1,60 @@
+name: Actuated aarch64 test
+
+on: [push, pull_request]
+
+# Cancel any preceding run on the pull request.
+concurrency:
+  group: stream-test-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/criu-dev' }}
+
+jobs:
+  build:
+    runs-on: actuated-arm64-4cpu-3gb
+    strategy:
+      matrix:
+        target: [GCC=1, CLANG=1]
+
+    steps:
+    # https://gist.github.com/alexellis/1f33e581c75e11e161fe613c46180771#file-metering-gha-md
+    # vmmeter start
+    - name: Prepare arkade
+      uses: alexellis/arkade-get@master
+      with:
+        crane: latest
+        print-summary: false
+
+    - name: Install vmmeter
+      run: |
+        crane export --platform linux/arm64 ghcr.io/openfaasltd/vmmeter:latest | sudo tar -xvf - -C /usr/local/bin
+
+    - name: Run vmmeter
+      uses: self-actuated/vmmeter-action@master
+    # vmmeter end
+
+    - uses: actions/checkout@v4
+    - name: Run Tests ${{ matrix.target }}
+      # Following tests are failing on the actuated VMs:
+      # ################### 3 TEST(S) FAILED (TOTAL 463/SKIPPED 54) ####################
+      # * zdtm/static/change_mnt_context(ns)
+      # * zdtm/static/macvlan(ns)
+      # * zdtm/static/sched_policy00(ns)
+      ###################################### FAIL #####################################
+      #
+      #    ./change_mnt_context --pidfile=change_mnt_context.pid --outfile=change_mnt_context.out
+      #    45: ERR: change_mnt_context.c:23: mount (errno = 22 (Invalid argument))
```

Comment:
It looks like the tests are running in an environment based on Ubuntu 22.04 with SELinux.
https://wiki.ubuntu.com/SELinux

```
2024-07-06T08:16:07.2924608Z ############### Distribution information ###############
2024-07-06T08:16:07.2925191Z DISTRIB_ID=Ubuntu
2024-07-06T08:16:07.2930731Z DISTRIB_RELEASE=22.04
2024-07-06T08:16:07.2931187Z DISTRIB_CODENAME=jammy
2024-07-06T08:16:07.2932000Z DISTRIB_DESCRIPTION="Ubuntu 22.04.4 LTS"
```
```
2024-07-06T08:19:37.3173083Z ===================== Test zdtm/static/seccomp_filter PASS =====================
2024-07-06T08:19:37.3173624Z 
2024-07-06T08:19:37.3174286Z === Run 244/463 ========-------- zdtm/static/seccomp_filter_inheritance
2024-07-06T08:19:38.6677896Z seccomp_suspend is supported
2024-07-06T08:19:40.1274685Z zdtm/static/selinux00.checkskip: line 24: setenforce: command not found
2024-07-06T08:19:40.1324588Z zdtm/static/selinux01.checkskip: line 24: setenforce: command not found
2024-07-06T08:19:40.1372864Z zdtm/static/selinux02.checkskip: line 24: setenforce: command not found
```

Reply:
enhanced the checkskip script for selinux00